examples/gallery/lines/contours.py

```diff
@@ -0,0 +1,41 @@
+"""
+Contours
+--------
+The :meth:`pygmt.Figure.contour` method can plot contour lines from a table of points by direct triangulation.
+The data to the triangulation can by provided in one of three options:
+    1: ``x``, ``y``, ``z`` 1d :class:`numpy.ndarray` data columns\n
+    2: ``data`` 2d :class:`numpy.ndarray` data matrix with 3 columns corresponding to ``x``, ``y``, ``z``\n
+    3: ``data`` path string to a file containing the ``x``, ``y``, ``z`` in a tabular format\n
+The parameters ``levels`` and ``annotation`` are deciding on the contours intervals and intervals of the
+annotation on the contours respectively.\n
+In this example we supply the data as  1d :class:`numpy.ndarray` with the ``x``, ``y``,
+and ``z`` parameters and draw the contours using a 0.5p pen with contours every 5 ``z`` values and
```

Comment:
```suggestion
and ``z`` parameters and draw the contours using a 0.5p pen with contours every 10 ``z`` values and
```
I think it's a bit easier to see the difference between the contour spacing and annotation spacing if the contours are slightly farther apart, so I am suggestive a contour level of 10 rather than 5.